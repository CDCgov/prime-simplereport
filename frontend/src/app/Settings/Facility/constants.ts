const isRequiredErrMsg = (fieldIdentifier: string) => {
  return `${fieldIdentifier} is required`;
};

const isInvalidErrMsg = (fieldIdentifier: string) => {
  return `${fieldIdentifier} is invalid`;
};

const facilityInfoLabels = {
  name: "Facility name",
  phone: "Facility phone number",
  email: "Email",
  street: "Facility street",
  zip: "Facility ZIP code",
  state: "Facility state",
  clia: "Facility CLIA number",
};

const orderingProviderLabels = {
  first: "Ordering provider first name",
  last: "Ordering provider last name",
  phone: "Ordering provider phone number",
  zip: "Ordering provider ZIP code",
};

export const facilityInfoErrMsgs = {
  name: {
    required: isRequiredErrMsg(facilityInfoLabels.name),
  },
  phone: {
    required: isRequiredErrMsg(facilityInfoLabels.phone),
    invalid: isInvalidErrMsg(facilityInfoLabels.phone),
  },
  email: {
    invalid: `${facilityInfoLabels.email} is incorrectly formatted`,
  },
  street: {
    required: isRequiredErrMsg(facilityInfoLabels.street),
  },
  zip: {
    required: isRequiredErrMsg(facilityInfoLabels.zip),
    invalid: isInvalidErrMsg(facilityInfoLabels.zip),
    stateInvalid: "Invalid ZIP code for this state",
  },
  state: {
    required: isRequiredErrMsg(facilityInfoLabels.state),
    invalid: "SimpleReport isnâ€™t currently supported in",
  },
  clia: {
    required: isRequiredErrMsg(facilityInfoLabels.clia),
    invalid:
      "CLIA numbers must be 10 characters (##D#######), or a special temporary number from CA, IL, VT, WA, WY, or the Department of Defense",
  },
};

export const orderingProviderErrMsgs = {
  first: {
    required: isRequiredErrMsg(orderingProviderLabels.first),
  },
  last: {
    required: isRequiredErrMsg(orderingProviderLabels.last),
  },
  npi: {
    required: "NPI should be a 10-digit numerical value (##########)",
  },
  phone: {
    required: isRequiredErrMsg(orderingProviderLabels.phone),
    invalid: isInvalidErrMsg(orderingProviderLabels.phone),
  },
  zip: {
    invalid: isInvalidErrMsg(orderingProviderLabels.zip),
  },
};

export const deviceRequiredErrMsg = "There must be at least one device";
