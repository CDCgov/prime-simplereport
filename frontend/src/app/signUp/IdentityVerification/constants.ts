export const exampleQuestionSet = [
  {
    questionType: 28,
    questionText:
      "Please select the model year of the vehicle you purchased or leased prior to January 2011 .",
    questionSelect: {
      questionChoice: [
        "2002",
        "2003",
        "2004",
        "2005",
        "NONE OF THE ABOVE/DOES NOT APPLY",
      ],
    },
  },
  {
    questionType: 24,
    questionText:
      "Which of the following professions do you currently have, or if there is not a matched profession, please select 'NONE OF THE ABOVE'.",
    questionSelect: {
      questionChoice: [
        "DENTIST / DENTAL HYGIENIST",
        "SOCIAL WORKER",
        "OPTICIAN / OPTOMETRIST",
        "ELECTRICIAN",
        "NONE OF THE ABOVE/DOES NOT APPLY",
      ],
    },
  },
  {
    questionType: 41,
    questionText:
      "Please select the number of bedrooms in your home from the following choices. If the number of bedrooms in your home is not one of the choices please select 'NONE OF THE ABOVE'.",
    questionSelect: {
      questionChoice: [
        "TWO",
        "THREE",
        "FOUR",
        "FIVE",
        "NONE OF THE ABOVE/DOES NOT APPLY",
      ],
    },
  },
  {
    questionType: 47,
    questionText:
      "According to your credit profile, you may have opened a Home type loan in or around May 2015. Please select the lender to whom you currently made your payments.",
    questionSelect: {
      questionChoice: [
        "UC LENDING",
        "CTX MORTGAGE",
        "MID AMERICA MORTGAGE",
        "1ST NATIONWIDE MTG",
        "NONE OF THE ABOVE/DOES NOT APPLY",
      ],
    },
  },
  {
    questionType: 49,
    questionText:
      "According to our records, you graduated from which of the following High Schools?",
    questionSelect: {
      questionChoice: [
        "BAXTER SPRINGS HIGH SCHOOL",
        "BELLS HIGH SCHOOL",
        "LOCKNEY HIGH SCHOOL",
        "AGUA DULCE HIGH SCHOOL",
        "NONE OF THE ABOVE/DOES NOT APPLY",
      ],
    },
  },
];
