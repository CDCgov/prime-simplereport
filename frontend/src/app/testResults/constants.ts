export enum MULTIPLEX_DISEASES {
  COVID_19 = "COVID-19",
  FLU_A = "Flu A",
  FLU_B = "Flu B",
}

export enum TEST_RESULTS {
  POSITIVE = "POSITIVE",
  NEGATIVE = "NEGATIVE",
  UNDETERMINED = "UNDETERMINED",
  UNKNOWN = "UNKNOWN",
}
