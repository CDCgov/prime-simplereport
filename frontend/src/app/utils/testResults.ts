function getTestResult(result: MultiplexResult): TestResult {
  if (result) {
    if ("testResult" in result) {
      return result.testResult;
    }
  }
  return "UNKNOWN";
}

export function getResultObjByDiseaseName(
  results: MultiplexResults,
  diseaseName: MultiplexDisease
): MultiplexResult {
  return (
    (results.find((result: MultiplexResult) => {
      return result.disease.name.includes(diseaseName);
    }) as MultiplexResult) || null
  );
}

export function getResultByDiseaseName(
  results: MultiplexResults,
  diseaseName: MultiplexDisease
): string {
  const result = getResultObjByDiseaseName(results, diseaseName);
  return getTestResult(result) || "UNKNOWN";
}

export function getSortedResults(results: MultiplexResults): MultiplexResults {
  return Object.values(results).sort(
    (a: MultiplexResult, b: MultiplexResult) => {
      return a.disease.name.localeCompare(b.disease.name);
    }
  );
}

export function hasMultiplexResults(results: MultiplexResults): boolean {
  return results?.length > 1
    ? results.some(
        (result: MultiplexResult) => result.disease?.name !== "COVID-19"
      )
    : false;
}

export function hasPositiveFluResults(results: MultiplexResults): boolean {
  return (
    results.filter(
      (multiplexResult: MultiplexResult) =>
        multiplexResult.disease.name.includes("Flu") &&
        getTestResult(multiplexResult) === "POSITIVE"
    ).length > 0
  );
}
