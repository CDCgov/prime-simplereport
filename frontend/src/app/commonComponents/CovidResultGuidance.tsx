import React from "react";
import { Trans, useTranslation } from "react-i18next";

import { COVID_RESULTS } from "../constants";

interface Props {
  result: string;
  isPatientApp: boolean;
}

const CovidResultGuidance = (props: Props) => {
  const result = props.result;
  const isPatientApp = props.isPatientApp;
  const { t } = useTranslation();

  switch (result) {
    case COVID_RESULTS.POSITIVE:
      return (
        <>
          <p>{t("testResult.notes.positive.p1")}</p>
          <ul>
            <li>{t("testResult.notes.positive.guidelines.li0")}</li>
            <li>{t("testResult.notes.positive.guidelines.li1")}</li>
            <li>{t("testResult.notes.positive.guidelines.li2")}</li>
            <li>{t("testResult.notes.positive.guidelines.li3")}</li>
            <li>{t("testResult.notes.positive.guidelines.li4")}</li>
            <li>{t("testResult.notes.positive.guidelines.li5")}</li>
          </ul>
          {isPatientApp ? (
            <Trans
              t={t}
              parent="p"
              i18nKey="testResult.notes.positive.p2"
              components={[
                <a href="https://www.cdc.gov/coronavirus/2019-ncov/symptoms-testing/symptoms.html">
                  Watch for symptoms and learn when to seek emergency medical
                  attention
                </a>,
              ]}
            />
          ) : (
            <Trans
              t={t}
              parent="p"
              i18nKey="testResult.notes.positive.p2"
              components={[
                <a
                  href={t("testResult.notes.positive.symptomsLink")}
                  target="_blank"
                  rel="noopener noreferrer"
                >
                  symptoms link
                </a>,
              ]}
            />
          )}
          <ul>
            <li>{t("testResult.notes.positive.emergency.li0")}</li>
            <li>{t("testResult.notes.positive.emergency.li1")}</li>
            <li>{t("testResult.notes.positive.emergency.li2")}</li>
            <li>{t("testResult.notes.positive.emergency.li3")}</li>
            <li>{t("testResult.notes.positive.emergency.li4")}</li>
          </ul>
          <p>{t("testResult.notes.positive.p3")}</p>
        </>
      );
    case COVID_RESULTS.NEGATIVE:
      return (
        <>
          <p>{t("testResult.notes.negative.p0")}</p>
          <ul className={isPatientApp ? "" : "sr-multi-column"}>
            <li>{t("testResult.notes.negative.symptoms.li0")}</li>
            <li>{t("testResult.notes.negative.symptoms.li1")}</li>
            <li>{t("testResult.notes.negative.symptoms.li2")}</li>
            <li>{t("testResult.notes.negative.symptoms.li3")}</li>
            <li>{t("testResult.notes.negative.symptoms.li4")}</li>
            <li>{t("testResult.notes.negative.symptoms.li5")}</li>
            <li>{t("testResult.notes.negative.symptoms.li6")}</li>
            <li>{t("testResult.notes.negative.symptoms.li7")}</li>
            <li>{t("testResult.notes.negative.symptoms.li8")}</li>
            <li>{t("testResult.notes.negative.symptoms.li9")}</li>
            <li>{t("testResult.notes.negative.symptoms.li10")}</li>
          </ul>
        </>
      );
    default:
      return (
        <>
          <p>{t("testResult.notes.inconclusive.p0")}</p>
          {isPatientApp && <p>{t("testResult.notes.inconclusive.p1")}</p>}
        </>
      );
  }
};

export default CovidResultGuidance;
