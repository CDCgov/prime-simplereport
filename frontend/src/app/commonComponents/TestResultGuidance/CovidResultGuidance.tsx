import React from "react";
import { Trans, useTranslation } from "react-i18next";

import { TEST_RESULTS } from "../../testResults/constants";

type CovidResultInfoProps = {
  result: TestResult;
  isPatientApp: boolean;
};

const CovidResultInfo = ({ result, isPatientApp }: CovidResultInfoProps) => {
  const { t } = useTranslation();

  switch (result) {
    case TEST_RESULTS.POSITIVE:
      return (
        <>
          <p>{t("testResult.notes.positive.p1")}</p>
          <ul>
            <li>{t("testResult.notes.positive.guidelines.li0")}</li>
            <li>{t("testResult.notes.positive.guidelines.li1")}</li>
            <li>{t("testResult.notes.positive.guidelines.li2")}</li>
            <li>{t("testResult.notes.positive.guidelines.li3")}</li>
            <li>{t("testResult.notes.positive.guidelines.li4")}</li>
            <li>{t("testResult.notes.positive.guidelines.li5")}</li>
          </ul>
          <Trans
            parent="p"
            i18nKey="testResult.notes.positive.p2"
            components={[
              <a
                href={t("testResult.notes.positive.symptomsLink")}
                target="_blank"
                rel="noopener noreferrer"
                key="symptoms-link"
              >
                symptoms link
              </a>,
            ]}
          />
          <ul>
            <li>{t("testResult.notes.positive.emergency.li0")}</li>
            <li>{t("testResult.notes.positive.emergency.li1")}</li>
            <li>{t("testResult.notes.positive.emergency.li2")}</li>
            <li>{t("testResult.notes.positive.emergency.li3")}</li>
            <li>{t("testResult.notes.positive.emergency.li4")}</li>
          </ul>
          <p>{t("testResult.notes.positive.p3")}</p>
          <Trans
            parent="p"
            i18nKey="testResult.information"
            components={[
              <a
                href={t("testResult.cdcLink")}
                target="_blank"
                rel="noopener noreferrer"
                key="cdc-link-one"
              >
                cdc.gov
              </a>,
              <a
                href={t("testResult.countyCheckToolLink")}
                target="_blank"
                rel="noopener noreferrer"
                key="county-check-tool-link"
              >
                county check tool
              </a>,
            ]}
          />
        </>
      );
    case TEST_RESULTS.NEGATIVE:
      return (
        <>
          <p>{t("testResult.notes.negative.p0")}</p>
          <ul className={isPatientApp ? "" : "sr-multi-column"}>
            <li>{t("testResult.notes.negative.symptoms.li0")}</li>
            <li>{t("testResult.notes.negative.symptoms.li1")}</li>
            <li>{t("testResult.notes.negative.symptoms.li2")}</li>
            <li>{t("testResult.notes.negative.symptoms.li3")}</li>
            <li>{t("testResult.notes.negative.symptoms.li4")}</li>
            <li>{t("testResult.notes.negative.symptoms.li5")}</li>
            <li>{t("testResult.notes.negative.symptoms.li6")}</li>
            <li>{t("testResult.notes.negative.symptoms.li7")}</li>
            <li>{t("testResult.notes.negative.symptoms.li8")}</li>
            <li>{t("testResult.notes.negative.symptoms.li9")}</li>
            <li>{t("testResult.notes.negative.symptoms.li10")}</li>
          </ul>
          <Trans
            parent="p"
            i18nKey="testResult.notes.negative.moreInformation"
            components={[
              <a
                href={t("testResult.cdcLink")}
                target="_blank"
                rel="noopener noreferrer"
                key="cdc-link-two"
              >
                cdc.gov
              </a>,
            ]}
          />
        </>
      );
    case TEST_RESULTS.UNDETERMINED:
      return (
        <>
          <p>{t("testResult.notes.inconclusive.p0")}</p>
          {isPatientApp && <p>{t("testResult.notes.inconclusive.p1")}</p>}
        </>
      );
    default:
      return null;
  }
};
interface CovidResultGuidanceProps {
  result: MultiplexResult;
  isPatientApp: boolean;
}
const CovidResultGuidance = ({
  result,
  isPatientApp,
}: CovidResultGuidanceProps) => {
  const { t } = useTranslation();

  return (
    <>
      <p className="text-bold sr-guidance-heading">
        {t("testResult.notes.h1")}
      </p>
      <CovidResultInfo result={result.testResult} isPatientApp={isPatientApp} />
    </>
  );
};
export default CovidResultGuidance;
