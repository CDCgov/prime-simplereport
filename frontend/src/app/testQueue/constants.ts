import { AlertContent } from "../commonComponents/Alert";
import { displayFullName } from "../utils";

export const QUEUE_NOTIFICATION_TYPES = {
  ADDED_TO_QUEUE__SUCCESS: 1,
  SUBMITTED_RESULT__SUCCESS: 2,
  DELIVERED_RESULT_TO_PATIENT__FAILURE: 3,
};

export type SomeoneWithName = {
  firstName?: string | null;
  middleName?: string | null;
  lastName?: string | null;
};

export const ALERT_CONTENT = {
  [QUEUE_NOTIFICATION_TYPES.ADDED_TO_QUEUE__SUCCESS]: <
    T extends SomeoneWithName
  >(
    patient: T,
    startWithLastName: boolean = true
  ): AlertContent => {
    return {
      type: "success",
      title: `${displayFullName(
        patient.firstName,
        patient.middleName,
        patient.lastName,
        startWithLastName
      )} was added to the queue`,
      body: "Newly added patients go to the bottom of the queue",
    };
  },
  [QUEUE_NOTIFICATION_TYPES.SUBMITTED_RESULT__SUCCESS]: <
    T extends SomeoneWithName
  >(
    patient: T,
    startWithLastName: boolean = true,
    dataRetentionLimitsEnabled: boolean = false
  ): AlertContent => {
    const resultText = dataRetentionLimitsEnabled
      ? "has been sent."
      : "was saved and reported.";
    return {
      type: "success",
      title: `Result for ${displayFullName(
        patient.firstName,
        patient.middleName,
        patient.lastName,
        startWithLastName
      )} ${resultText}`,
      body: "See Results to view all test submissions",
    };
  },
  [QUEUE_NOTIFICATION_TYPES.DELIVERED_RESULT_TO_PATIENT__FAILURE]: <
    T extends SomeoneWithName
  >(
    patient: T,
    startWithLastName: boolean = true
  ): AlertContent => {
    return {
      type: "error",
      title: `Unable to text result to ${displayFullName(
        patient.firstName,
        patient.middleName,
        patient.lastName,
        startWithLastName
      )}`,
      body: "The phone number provided may not be valid or may not be able to accept text messages",
    };
  },
};

export const MIN_SEARCH_CHARACTER_COUNT = 2;

export const SEARCH_DEBOUNCE_TIME = 500;
