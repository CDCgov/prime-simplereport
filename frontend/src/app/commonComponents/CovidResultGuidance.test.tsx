import { render } from "@testing-library/react";

import { TEST_RESULTS } from "../testResults/constants";

import CovidResultGuidance from "./CovidResultGuidance";
import "../../i18n";

describe("CovidResultGuidance", () => {
  const cases = [
    TEST_RESULTS.POSITIVE,
    TEST_RESULTS.NEGATIVE,
    TEST_RESULTS.UNDETERMINED,
    TEST_RESULTS.UNKNOWN,
  ];
  test.each(cases)("displays guidance for %p result", (testResult) => {
    const { container: containerReporting } = render(
      <CovidResultGuidance
        result={testResult}
        isPatientApp={false}
        needsHeading={true}
      />
    );

    expect(containerReporting).toMatchSnapshot();
    const { container: containerPXP } = render(
      <CovidResultGuidance
        result={testResult}
        isPatientApp={true}
        needsHeading={false}
      />
    );
    expect(containerPXP).toMatchSnapshot();
  });
});
