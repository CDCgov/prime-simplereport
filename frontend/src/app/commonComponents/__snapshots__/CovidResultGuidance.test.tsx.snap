// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CovidResultGuidance COVID-19 result guidance displays guidance for "NEGATIVE" result 1`] = `
<div>
  <p
    class="text-bold sr-guidance-heading"
  >
    testResult.notes.h1
  </p>
  <p>
    testResult.notes.negative.p0
  </p>
  <ul
    class="sr-multi-column"
  >
    <li>
      testResult.notes.negative.symptoms.li0
    </li>
    <li>
      testResult.notes.negative.symptoms.li1
    </li>
    <li>
      testResult.notes.negative.symptoms.li2
    </li>
    <li>
      testResult.notes.negative.symptoms.li3
    </li>
    <li>
      testResult.notes.negative.symptoms.li4
    </li>
    <li>
      testResult.notes.negative.symptoms.li5
    </li>
    <li>
      testResult.notes.negative.symptoms.li6
    </li>
    <li>
      testResult.notes.negative.symptoms.li7
    </li>
    <li>
      testResult.notes.negative.symptoms.li8
    </li>
    <li>
      testResult.notes.negative.symptoms.li9
    </li>
    <li>
      testResult.notes.negative.symptoms.li10
    </li>
  </ul>
</div>
`;

exports[`CovidResultGuidance COVID-19 result guidance displays guidance for "NEGATIVE" result 2`] = `
<div>
  <p>
    testResult.notes.negative.p0
  </p>
  <ul
    class=""
  >
    <li>
      testResult.notes.negative.symptoms.li0
    </li>
    <li>
      testResult.notes.negative.symptoms.li1
    </li>
    <li>
      testResult.notes.negative.symptoms.li2
    </li>
    <li>
      testResult.notes.negative.symptoms.li3
    </li>
    <li>
      testResult.notes.negative.symptoms.li4
    </li>
    <li>
      testResult.notes.negative.symptoms.li5
    </li>
    <li>
      testResult.notes.negative.symptoms.li6
    </li>
    <li>
      testResult.notes.negative.symptoms.li7
    </li>
    <li>
      testResult.notes.negative.symptoms.li8
    </li>
    <li>
      testResult.notes.negative.symptoms.li9
    </li>
    <li>
      testResult.notes.negative.symptoms.li10
    </li>
  </ul>
</div>
`;

exports[`CovidResultGuidance COVID-19 result guidance displays guidance for "POSITIVE" result 1`] = `
<div>
  <p
    class="text-bold sr-guidance-heading"
  >
    testResult.notes.h1
  </p>
  <p>
    testResult.notes.positive.p1
  </p>
  <ul>
    <li>
      testResult.notes.positive.guidelines.li0
    </li>
    <li>
      testResult.notes.positive.guidelines.li1
    </li>
    <li>
      testResult.notes.positive.guidelines.li2
    </li>
    <li>
      testResult.notes.positive.guidelines.li3
    </li>
    <li>
      testResult.notes.positive.guidelines.li4
    </li>
    <li>
      testResult.notes.positive.guidelines.li5
    </li>
  </ul>
  <ul>
    <li>
      testResult.notes.positive.emergency.li0
    </li>
    <li>
      testResult.notes.positive.emergency.li1
    </li>
    <li>
      testResult.notes.positive.emergency.li2
    </li>
    <li>
      testResult.notes.positive.emergency.li3
    </li>
    <li>
      testResult.notes.positive.emergency.li4
    </li>
  </ul>
  <p>
    testResult.notes.positive.p3
  </p>
</div>
`;

exports[`CovidResultGuidance COVID-19 result guidance displays guidance for "POSITIVE" result 2`] = `
<div>
  <p>
    testResult.notes.positive.p1
  </p>
  <ul>
    <li>
      testResult.notes.positive.guidelines.li0
    </li>
    <li>
      testResult.notes.positive.guidelines.li1
    </li>
    <li>
      testResult.notes.positive.guidelines.li2
    </li>
    <li>
      testResult.notes.positive.guidelines.li3
    </li>
    <li>
      testResult.notes.positive.guidelines.li4
    </li>
    <li>
      testResult.notes.positive.guidelines.li5
    </li>
  </ul>
  <ul>
    <li>
      testResult.notes.positive.emergency.li0
    </li>
    <li>
      testResult.notes.positive.emergency.li1
    </li>
    <li>
      testResult.notes.positive.emergency.li2
    </li>
    <li>
      testResult.notes.positive.emergency.li3
    </li>
    <li>
      testResult.notes.positive.emergency.li4
    </li>
  </ul>
  <p>
    testResult.notes.positive.p3
  </p>
</div>
`;

exports[`CovidResultGuidance COVID-19 result guidance displays guidance for "UNDETERMINED" result 1`] = `
<div>
  <p
    class="text-bold sr-guidance-heading"
  >
    testResult.notes.h1
  </p>
  <p>
    testResult.notes.inconclusive.p0
  </p>
</div>
`;

exports[`CovidResultGuidance COVID-19 result guidance displays guidance for "UNDETERMINED" result 2`] = `
<div>
  <p>
    testResult.notes.inconclusive.p0
  </p>
  <p>
    testResult.notes.inconclusive.p1
  </p>
</div>
`;

exports[`CovidResultGuidance COVID-19 result guidance displays guidance for "UNKNOWN" result 1`] = `
<div>
  <p
    class="text-bold sr-guidance-heading"
  >
    testResult.notes.h1
  </p>
</div>
`;

exports[`CovidResultGuidance COVID-19 result guidance displays guidance for "UNKNOWN" result 2`] = `<div />`;
